testutils/utils.go

```diff
@@ -334,11 +335,23 @@ func RunCNIPluginWithId(netconf, podName, podNamespace, ip, containerId, ifName
 }
 
 // Create veth pair on host
-func CreateHostVeth(containerId string, k8sName string, k8sNamespace string) error {
+func CreateHostVeth(containerId, k8sName, k8sNamespace, nodename string) error {
```

Comment:
Kinda feels like this should take a WorkloadEndpointIdentifiers struct as the parm.

Reply:
not expecting you to change this though... just commenting.